plugins/00_dokku-standard/commands

```diff
@@ -65,15 +66,21 @@ case "$1" in
         # *DEPRECATED* in v0.3.22: `pluginhook pre-release-buildstep` will be removed in future releases
         pluginhook pre-release-buildstep "$APP"
         pluginhook pre-release-buildpack "$APP"
-        if [[ -f "$DOKKU_ROOT/ENV" ]]; then
-          id=$(docker run -i -a stdin $IMAGE /bin/bash -c "mkdir -p /app/.profile.d && cat > /app/.profile.d/00-global-env.sh" < "$DOKKU_ROOT/ENV")
+        if [[ -n $(config_export global) ]]; then
+          TMPFILE=$(mktemp /tmp/dokku.XXXXXX) || dokku_log_fail "Cannot create temp file using mktemp in /tmp dir"
+          config_export global > $TMPFILE
+          id=$(docker run -i -a stdin $IMAGE /bin/bash -c "mkdir -p /app/.profile.d && cat > /app/.profile.d/00-global-env.sh" < "$TMPFILE")
           test "$(docker wait $id)" -eq 0
           docker commit $id $IMAGE > /dev/null
+          rm -rf $TMPFILE
```

Comment:
We do similar things in the git plugin and main line even.

https://github.com/progrium/dokku/blob/4e0fed18a6f93eef4801f6c67312e7eae6bf35ac/plugins/git/commands#L11
https://github.com/progrium/dokku/blob/ae2e0bc57c93bbdf07cf676097af5c49957cccdf/dokku#L139